docs/user-docs/latest/content-types-builder/configuring-fields-content-type.md

```diff
@@ -0,0 +1,410 @@
+
+:::
+
+:::: 
+
+### <img width="28" src="../assets/content-types-builder/field-icon_email.png"> Email
+
+The Email field displays an email address field with format validation to ensure the email address is valid.
+
+:::: tabs
+
+::: tab Base settings
+
+| Setting name  | Instructions                                                    |
+|---------------|-----------------------------------------------------------------|
+| Name          | Write the name of the Email field.                              |
+
+:::
+
+::: tab Advanced settings
+
+| Setting name   | Instructions                                                                |
+|----------------|-----------------------------------------------------------------------------|
+| Default value  | Write the default value of the Email field.                                 |
+| Private field  | Tick to make the field private and prevent it from being found via the API. |
+| Required field | Tick to prevent creating or saving an entry if the field is not filled in.  |
+| Unique field   | Tick to prevent another field to be identical to this one.                  |
+| Maximum length | Tick to define a maximum number of characters allowed.                      |
+| Minimum length | Tick to define a minimum number of characters allowed.                      |
+
+:::
+
+::::
+
+### <img width="28" src="../assets/content-types-builder/field-icon_password.png"> Password
+
+The Password field displays a password field that is encrypted.
+
+:::: tabs
+
+::: tab Base settings
+
+| Setting name  | Instructions                                                    |
+|---------------|-----------------------------------------------------------------|
+| Name          | Write the name of the Password field.                           |
+
+:::
+
+::: tab Advanced settings
+
+| Setting name   | Instructions                                                                |
+|----------------|-----------------------------------------------------------------------------|
+| Default value  | Write the default value of the Password field.                              |
+| Private field  | Tick to make the field private and prevent it from being found via the API. |
+| Required field | Tick to prevent creating or saving an entry if the field is not filled in.  |
+| Unique field   | Tick to prevent another field to be identical to this one.                  |
+| Maximum length | Tick to define a maximum number of characters allowed.                      |
+| Minimum length | Tick to define a minimum number of characters allowed.                      |
+
+:::
+
+::::
+
+### <img width="28" src="../assets/content-types-builder/field-icon_enum.png"> Enumeration
+
+The Enumeration field allows to configure a list of values displayed in a drop-down list.
```

Comment:
Might be good to add a link to the enumeration tracking issue about certain aspects (such as recommending a relation using either oneWay or manyWay for some use-cases instead of enum)

https://github.com/strapi/strapi/issues/7904

@meganelacheny I can help write a summery of this issue as it's quite large.

Reply:
Yes if there are some known issues & workarounds it could be a good idea to add a note about them. A summery would be very helpful @derrickmehaffy thank you!